ui/lib/UpdateStream.tsx

```diff
@@ -1,7 +1,224 @@
-import * as React from "react";
+import React, { useState, useEffect } from "react";
+import StreamManager, { Stream } from "../../sdk/js/index";
+import {
+  usePrepareContractWrite,
+  useContractWrite,
+  useAccount,
+  useBalance,
+} from "wagmi";
+import Slider from "rc-slider";
 
-const UpdateStream = () => {
-  return <div />;
+interface UpdateStreamProps {
+  stream: Stream;
+  streamDailyCost: number;
```

Comment:
question: Similar to my other question above.  Is `number` appropriate here?  Should this be `BigInt` if we're dealing with indivisible amounts?